include/klee/Expr.h

```diff
@@ -1080,7 +1080,8 @@ class ConstantExpr : public Expr {
   }
 
   static ref<ConstantExpr> create(uint64_t v, Width w) {
-    assert(v == bits64::truncateToNBits(v, w) && "invalid constant");
+    if (w <= 64)
```

Comment:
Oh I see. Well, I guess the compiler removes the `if` in case of build without debugging info. Anyways, the `#ifndef` is good.